ledger-api/grpc-definitions/BUILD.bazel

```diff
@@ -70,69 +73,115 @@ proto_gen(
     ],
 )
 
-# TODO(nic): When compile-proto-file handles map<key,value>, we will can run on:  **/*.proto
-# + glob(["**/*.proto",])
+google_protobuf_src = "external/com_google_protobuf/src"
+
+genrule(
+    name = "google-protobuf-haskellpb-sources",
+    srcs = ["@com_google_protobuf//:well_known_protos"],
+    outs = ["Google/Protobuf/" + b for b in [
+        "Any.hs",
+        "Duration.hs",
+        "Empty.hs",
+        "Timestamp.hs",
+        "Wrappers.hs",
+    ]],
+    cmd = """
+        for src in \
+            external/com_google_protobuf/src/google/protobuf/any.proto \
+            external/com_google_protobuf/src/google/protobuf/duration.proto \
+            external/com_google_protobuf/src/google/protobuf/empty.proto \
+            external/com_google_protobuf/src/google/protobuf/timestamp.proto \
+            external/com_google_protobuf/src/google/protobuf/wrappers.proto \
+        ; do
+            $(location @haskell_proto3__suite//:compile-proto-file) \
+                --includeDir """ + google_protobuf_src + """ \
```

Comment:
[`string.format`](https://docs.bazel.build/versions/master/skylark/lib/string.html#format) can be nice for this kind of stuff although for this particular case there is not a huge difference.

Reply:
thanks, will bear in mind for the future